spec/components/page_list_component/view_spec.rb

```diff
@@ -49,5 +50,39 @@
     it "renders a move down link" do
       expect(page).to have_button("Move down")
     end
+
+    context "when conditions are enabled", feature_basic_routing: true do
+      let(:pages) do
+        [OpenStruct.new(id: 1, question_text: "What country do you live in?", routing_conditions:),
+         OpenStruct.new(id: 2, question_text: "What is your name?", routing_conditions:),
+         OpenStruct.new(id: 3, question_text: "What is your pet's name?", routing_conditions:)]
+      end
+
+      context "when there are no conditions" do
+        it "conditions section is blank" do
+          expect(page).not_to have_text("Question 1's route")
```

Comment:
Is there another way we could tag conditions eg. css to html structure? The reason I'm say that is that if for some reason the `t("page_conditions.condition_name")` changed then this test would still pass because the page doesn't contain this specific tests.

Reply:
I don't think there's a CSS/HTML way to do it (the markup is the same for the conditions as for the pages themselves), but can improve this by calling the translation directly instead of hard coding it

Reply:
Sure we could improve the tests by using the translations but the thing I'm trying to avoid here is our usage of `.not_to have...` because theres more chance of some not being on the page than there would be if it was expected on the page.

What if the test was to only have 1 question, we could check the page only has one `.govuk-summary-list__row` or actually in this test because there are three questions and they dont have any routes we would expect there to only be 3 `.govuk-summary-list__row` 

```
expect(page).to have_css('.govuk-summary-list__row',count: 3)
```

Reply:
Oh, I see what you mean - thought you were just looking for something more specific than the string for the selector. This does sound more robust, could probably even do:
```
expect(page).to have_css('.govuk-summary-list__row',count: pages.length)
```
to avoid the risk of the counts getting out of sync. Will try locally

Reply:
Done!